Shaders/MeshChain.cginc

```diff
@@ -1,5 +1,6 @@
 #ifndef MESH_CHAIN
 #define MESH_CHAIN
+// UNITY_SHADER_NO_UPGRADE
```

Comment:
Further in the file is a define that switches between the old method of vertex transformation and new method compatible with single-pass stereo.  Without that comment, Unity will upgrade the old method away, which is not what we want.